### PR TITLE
Replace cpx with cpx2 for version 2.19.x

### PR DESCRIPTION
**Pull request by evelynpreslar-bentley**
Mirroring [this PR](https://github.com/iTwin/itwinjs-core/pull/2714) for 2.19.x due to cpx vulnerability

### CODE REVIEW

File: docs/learning/WriteAConnector.md [hunk 1]
```diff
@@ -256,13 +256,13 @@ A Connector is usually dealing with two levels of provenance
 1. What is the identity and metadata of a file or repository synchronized into an iModel?
 2. What is the identity of the element within that repository?
 
-[RepositoryLink]($imodeljs-backend) is a specialization of a UrlLink which has a "Url" property pointing to an external resource or repository and a RepositoryGuid that identifies the external repository.
+[RepositoryLink]($core-backend) is a specialization of a UrlLink which has a "Url" property pointing to an external resource or repository and a RepositoryGuid that identifies the external repository.
```
Comment (calebmshafer) [inline docs/learning/WriteAConnector.md:259]: These package renames aren't going to work in this branch

Comment (evelynpreslar-bentley) [reply docs/learning/WriteAConnector.md]: Just reverted those, as well as the section under "Locks & Codes" which I'm assuming was also describing 3.0 behavior in the version I pushed.

File: docs/learning/WriteAConnector.md [hunk 2]
```diff
@@ -2,63 +2,63 @@
...
-  - [Building a test for a Connector](#building-a-test-for-a-connector)
-- [Advanced Topics](#advanced-topics)
-  - [Job Subjects](#job-subjects)
-  - [Schema merging](#schema-merging)
-  - [Units and Coordinate systems](#units-and-coordinate-systems)
-  - [Dealing with geometry](#dealing-with-geometry)
-  - [Authentication](#authentication)
-  - [Locks & Codes](#locks--codes)
-  - [More information](#more-information)
+- [Write A Connector](#write-a-connector)
```
Comment (calebmshafer) [inline docs/learning/WriteAConnector.md:5]: Where did this top level come from?

**Review by pmconne — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

File: docs/learning/WriteAConnector.md (hunk 2 above)
Comment (evelynpreslar-bentley) [reply docs/learning/WriteAConnector.md:5]: It was merged into my working copy of the file when I cherry-picked the commit, as were the package renames and changing `imodeljs` -> `itwinjs-core` in the SDK URL.

Want me to revert all but the `cpx` -> `cpx2` change or keep both that and the URL change?

**Review by aruniverse — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

**Review by ColinKerr — APPROVED** [review]

**Review by ramanujam-raman — APPROVED** [review]

**Comment by calebmshafer** [comment]
@evelynpreslar-bentley I'll override the rest of the reviews once the builds complete.

**Review by jchick-bentley — APPROVED** [review]

**Comment by calebmshafer** [comment]
@evelynpreslar-bentley looks like there may be something wrong with the windows build. Mind taking a look? I queued it a few times in a row to see if it was flaky and that doesn't seem to be the case.

**Comment by evelynpreslar-bentley** [comment]
@calebmshafer I noticed some localization file parsing errors that seemed to be causing the copyfile errors we see in the output, so I moved the NoRenderApp.startup call (where the localization parsing calls originate) to run after the relevant files are copied.
The failing tests seem to be resolved now.

The best I can guess is some async JSON parsing was interfering with the copying and making the test initialization fail.
I don't know why this only happened in cpx2 vs cpx, but I imagine that this didn't crop up in 3.x because it makes use of a new localization system.

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**